grype/vulnerability_matcher_test.go

```diff
@@ -552,11 +553,17 @@ func TestVulnerabilityMatcher_FindMatches(t *testing.T) {
 						Constraint: version.MustGetConstraint("< 3.7.6", version.UnknownFormat),
 						ID:         "CVE-2014-fake-3",
 						Namespace:  "nvd:cpe",
-						CPEs: []syftPkg.CPE{
+						CPEs: []cpe.CPE{
 							mustCPE("cpe:2.3:*:activerecord:activerecord:*:*:*:*:*:rails:*:*"),
 						},
 						PackageQualifiers: []qualifier.Qualifier{},
 						Advisories:        []vulnerability.Advisory{},
+						RelatedVulnerabilities: []vulnerability.Reference{
+							{
+								ID:        "GHSA-2014-fake-3",
+								Namespace: "github:language:ruby",
+							},
+						},
```

Comment:
This is actually one of the non-CPE changes with this PR: additional fields will be merged.